bundles/ranvier-quests/lib/BountyGoal.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const QuestGoal = require('../../../src/QuestGoal');
+
+/**
+ * A quest goal requiring the player to locate and/or return an NPC.
```

Comment:
This is super cool even if it doesn't address the original issue ;) I think it could be a worthy contribution to the core library.